styles/styles.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+body {
+  background-color: rgb(23, 23, 23);
+}
+
+.logo {
+  padding: 1.2rem;
+}
+
+.logo:hover {
+  color: #ff6b00;
+  cursor: pointer;
+}
```

Comment:
- [ ] Please import `Roboto` from [Google fonts](https://fonts.google.com/?query=roboto). Please make sure your fonts match with [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=rTgMr8v8dtgLyJg0-0)
- [ ] Also please make sure that you are using the correct font-size of the logo. 

_Note: Using fonts mentioned in the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=rTgMr8v8dtgLyJg0-0) is not necessary as sometimes Figma dimensions and real world dimensions are different, you have to increase the size to make sure your project looks almost identical to figma template_

**_Your design_**

<img width="270" alt="Screenshot 2023-02-17 at 12 07 18 AM" src="https://user-images.githubusercontent.com/44798044/219463792-6e4cddd5-4af6-495a-a4b5-3cde107ce507.png">

**_Figma Design_**

<img width="382" alt="Screenshot 2023-02-17 at 12 07 09 AM" src="https://user-images.githubusercontent.com/44798044/219463869-36971067-66e4-4e46-ba6f-2f3788b845fa.png">